department-of-veterans-affairs/va.gov-cms
# VACMS-8184 toolbar contrast

## Description

Closes #8184.

- Updates colors for various states in the top-level drupal toolbar to pass accessibility. 
- Updates the icon organization in our theme to match drupal core & core themes. icons are now sorted into subdirectories based on hexcode. Slightly easier to find the icon you are looking for in a given list.
- Removed a couple stale images I added early on that were not used. (different header logo, the CAG smartchip pic)

Main work is in the _toolbar.scss file, new mixin for consistency across the menu items with icons. Deleted old styles for menu links past, see this slack thread: https://dsva.slack.com/archives/CT4GZBM8F/p1646074826475409

## Testing done
- Lots of hovering, clicking between menu items, forcing hover, focus, active states on the top level menu items.
- Checked that icons appeared in the other places they should with the reorganization.

## Screenshots
Hover, active, and default states pictured:
<img width="961" alt="Screen Shot 2022-02-28 at 2 51 50 PM" src="https://user-images.githubusercontent.com/11279744/156072920-3af37b5f-82a6-49ed-84e2-845ebdb25262.png">
<img width="291" alt="Screen Shot 2022-02-28 at 2 52 16 PM" src="https://user-images.githubusercontent.com/11279744/156072924-d773facd-902f-4135-8b11-d9f52b595c27.png">

existing icons unchanged:
<img width="581" alt="Screen Shot 2022-02-28 at 2 51 33 PM" src="https://user-images.githubusercontent.com/11279744/156072918-b5331170-09fd-4622-9ba3-b5b9115d5829.png">
<img width="1172" alt="Screen Shot 2022-02-28 at 3 04 28 PM" src="https://user-images.githubusercontent.com/11279744/156073104-7fe16abb-61b6-403e-9afa-5448e5aeaadd.png">



## QA steps
enter wrong credentials on login, see that the error icon still shows
log in correctly
hover top level menu item, see that it passes a11y. click to open a menu and move mouse to see active state without hover

### Definition of Done


### Select Team for PR review


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
